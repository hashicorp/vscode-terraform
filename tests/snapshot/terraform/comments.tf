# line comment

// line comment

/*
  Block comment
*/
