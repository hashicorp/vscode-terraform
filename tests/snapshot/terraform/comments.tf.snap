># line comment
#^ source.terraform comment.line.terraform punctuation.definition.comment.terraform
# ^^^^^^^^^^^^^ source.terraform comment.line.terraform
>
>// line comment
#^^ source.terraform comment.line.terraform punctuation.definition.comment.terraform
#  ^^^^^^^^^^^^^ source.terraform comment.line.terraform
>
>/*
#^^ source.terraform comment.block.terraform punctuation.definition.comment.terraform
>  Block comment
#^^^^^^^^^^^^^^^^ source.terraform comment.block.terraform
>*/
#^^ source.terraform comment.block.terraform punctuation.definition.comment.terraform
>